### ISSUE TITLE
mega-login and quota

### ISSUE THREAD

**Issue by meeotch** [meganz/MEGAcmd#373]
Question:  If I log in using the folder syntax:

 mega-login 'https://mega.nz/#F!NGIVblahblahblahblah'

Is my megacmd session then using the quota associated with the account that hosts the folder?  Or is it using the quota associated with the IP address where the commands are being issued?

The context is:  I'm writing a script to synchronize some folders from my home server to a remote Synology NAS.  I want the Synology to log in to a specific mega folder (not to generally login with my user/pass), and download its contents.  I'd like the Synology to be using the quota of my primary mega account, so that the Synology and the home server don't have to each have their own mega accounts.

**Comment by polmr** [comment]
It will be using your granted transfer quota if you are a PRO user. This will help you: https://mega.nz/help/client/webclient/accounts-and-pro-accounts/when-do-i-need-to-upgrade-to-a-pro-account-576c7905886688e6028b4584

**Comment by meeotch** [comment]
Thanks for the explanation - that makes sense.  Can you confirm that I'm correctly understanding my options for transferring a given folder from my mega account (which I'll call primary@bob.com) to my remote Synology via mega-cmd?

1. login to the folder directly:  mega-login 'https://mega.nz/#F!NGIVblahblahblahblah'
This will use the quota of the primary@bob.com mega account hosting the folder.

2. no login:  mega-get 'https://mega.nz/#F!NGIVblahblahblahblah'
This will use the quota of the primary@bob.com mega account hosting the folder.

3. login to a separate (free) account:  mega-login synology@foo.com <password>, then mega-get 'https://mega.nz/#F!NGIVblahblahblahblah'
This will still use the quota of the primary@bob.com mega account hosting the folder.

4. share the folder from primary@bob.com to synology@foo.com, then mega-login synology@foo.com <password>, and mega-get 'foldername'
Not sure which transfer quota is used, but I'm guessing it will use the quota of the primary@bob.com mega account.  But it will only use disk quota on the primary@bob.com account.

5. import the folder from primary@bob.com to synology@foo.com, then mega-login synology@foo.com <password>, and mega-get 'foldername'
This will use the quota of the (free) synology@foo.com account - which is essentially the same as the (free) quota associated with the IP address where the Synology lives.  But it will also use the disk quota of the synology@foo.com account.

So basically, the only option that doesn't use 2x paid transfer quota (1x upload, 1x download, both on primary@bob.com) is the "import" option, using two separate mega accounts.
